### DISCUSSION TITLE
Clipboard trigger

### DISCUSSION THREAD

**Discussion by jtagcat** [espanso/espanso#1399] [Feature requests and Ideas]
Multiple times a day, I adjust text in my clipboard. Most often, it's cleaning up links in the similar ways.

It'd be neat if I could set triggers for content pasted (or copied), and edit the clipboard.

**Comment by kpym** [comment]
There is a [clipboard-extension](https://espanso.org/docs/matches/extensions/#clipboard-extension) that can be used in combination with a script that cleans the url.
